### DISCUSSION TITLE
What version of Postgres is Supabase hosting?

### DISCUSSION THREAD

**Discussion by nicholaschiang** [supabase/supabase#3990] [Questions] [answered]
# Improve documentation

## Link 

I could not find a page with this information, which is why this is an issue haha.

## Describe the problem

**What version(s) of Postgres is Supabase hosting?**

That ☝️ should be answered very clearly in the docs and IMO on [the product page](https://supabase.io/database).

## Describe the improvement

Specify the version(s) of Postgres that Supabase hosts.

**Comment by w3b6x9** [comment, marked as answer]
@nicholaschiang 

> What version(s) of Postgres is Supabase hosting?

v12.5

> That ☝️ should be answered very clearly in the docs and IMO on the product page.

Awesome feedback! You can get the version by running `SELECT version();` if you already have a Supabase Postgres db but totally agree that it should be spelled out somewhere like docs and/or product page for everyone's convenience. I'll pass this on to the team!

Thanks!

**Comment by w3b6x9** [comment]
@nicholaschiang also, feel free to contribute and tell us where you would like to see pg's version number! contributions always welcome!

**Comment by inian** [comment]
Hey @nicholaschiang, good point. We will show this in the dashboard itself since the postgres version might change in the future for new projects. You can also find the version number anytime by running `SELECT version();` in your SQL editor.

**Reply by brandonchinn178** [reply]
Is this shown anywhere yet? I don't see it under Settings or anywhere

**Comment by nicholaschiang** [comment]
@inian @w3b6x9 thanks! I'm totally new to PostgreSQL and relational database in general so I didn't know about `version()`.
